databuilder/models/dashboard_elasticsearch_document.py

```diff
@@ -8,13 +8,13 @@ class DashboardESDocument(ElasticsearchDocument):
     Schema for the ES dashboard ES document
     """
     def __init__(self,
-                 dashboard_group,  # type: str
-                 dashboard_name,  # type: str
+                 group_name,  # type: str
+                 name,  # type: str
```

Comment:
Shall we add `uri` as well?
https://github.com/lyft/amundsencommon/blob/master/amundsen_common/models/dashboard.py#L9

Reply:
uri is already there(L21)

Reply:
My bad, lol.